src/legacy/core_plugins/metrics/server/lib/vis_data/get_annotations.js

```diff
@@ -30,21 +31,22 @@ function validAnnotation(annotation) {
 
 export default async (req, panel) => {
   const { callWithRequest } = req.server.plugins.elasticsearch.getCluster('data');
+  const timeout = getEsShardTimeout(req);
   const bodies = panel.annotations
     .filter(validAnnotation)
     .map(annotation => {
-
       const indexPattern = annotation.index_pattern;
       const bodies = [];
+      const body = {
+        ...buildAnnotationRequest(req, panel, annotation),
+        timeout
+      };
 
       bodies.push({
```

Comment:
I see that  now you push into 'bodies' array 2 objects. What about to give more developer-friendly names 
e.g.
const header = {
   index: indexPattern,
   ignoreUnavailable: true,
};
bodies.push(header, body);